book-2-the-neophyte/chapters/DAILY_JOURNAL_MODULAR.md

```diff
@@ -53,18 +59,46 @@ Both are valid, but the first example saves a few characters.
 
 Now that you've defined an object whose responsibility is to access the data, you need to write code in `src/scripts/journal.js` to use that object and get the data. Once you know you have the data, pass it along to the `renderJournalEntries` function that now lives in `src/scripts/DOMPrinter.js`.
 
+`DOMPrinter.js:`
+```js
+import APIManager from './APIManager.js'
+
+const DomPrinter = {
+
+    renderJournalEntries () {
+        // method from APIManager that accesses the data goes here
+        ApiManager.getJournalEntries()
+        // Then you need to use the returned data
+        .then(returnedArray => {
+            // code that prints to the dom goes here
+
+        })
+
+
+    }
+
+}
+
+export default DOMPrinter
+```
+
+
 Put this comment in `src/scripts/journal.js`. Then write the main logic that uses the code in the helper modules.
 
 ```js
+import DOMPrinter from './DOMPrinter.js'
+
+
 /*
     Main application logic that uses the functions and objects
     defined in the other JavaScript files.
 
-    Change the fake variable names below to what they should be
+    call the method within DOMPrinter
     to get the data and display it.
 */
-objectWithGetterMethod.methodToGetData().then(functionThatRendersData)
+
 ```
+<!-- objectWithGetterMethod.methodToGetData().then(functionThatRendersData) -->
 
 ## Challenge
```

Comment:
I think there's still a challenge in there that doesn't make sense anymore (it has to do with turning them into objects, which I think they will have already done at this point). I think let's take the challenge out but otherwise looks great!